node/src/BaseClient.ts

```diff
@@ -3416,6 +3417,30 @@ export class BaseClient {
         );
     }
 
+    /**
+     * Returns the `GeoHash` strings representing the positions of all the specified `members` in the sorted set stored at `key`.
+     *
+     * See https://valkey.io/commands/geohash/ for more details.
+     *
+     * @param key - The key of the sorted set.
+     * @param members - The array of members whose <code>GeoHash</code> strings are to be retrieved.
+     * @returns An array of `GeoHash` strings representing the positions of the specified members stored at `key`.
+     *   If a member does not exist in the sorted set, a `null` value is returned for that member.
+     *
+     * @example
+     * ```typescript
+     * const num = await client.geohash("mySortedSet",["Palermo", "Catania", "NonExisting"]);
+     * console.log(num); // Output: prints a list of corresponding GeoHash String values.
```

Comment:
```suggestion
     * const result = await client.geohash("mySortedSet",["Palermo", "Catania", "NonExisting"]);
     * console.log(result ); // Output: [...]
```
Please show values, so user will see the `null`

Reply:
Copy from transaction IT